### PR TITLE
Space Travellers Hub

### PR DESCRIPTION
**Pull request by Oklukeok**
- [ ]  Setup Gitflow.
- [ ]  Installed react with npx create-react-app . command.
- [ ]  Installed and added linters.
- [ ]  Setup Kanban Boards.
- [ ]  Installed all dependencies including redux toolkit.
- [ ]  Add component structures.
- [ ]  Implemented functionality of fetching rockets.
- [ ]  Implemented fetching missions.
- [ ]  Implemented funtionality of reserving a rocket.
- [ ]  Implemented functionality of cancelling rocket reservation.
- [ ]  Implemented functionality of joining missions.
- [ ]  Implemented functionality of leaving missions.
- [ ]  Implemented displaying of reserved rockets and joined missions in My profile page.
- [ ]  Added styles for all the pages/components.
- [ ]  Tested all components if they render correctly.
- [ ]  Checked and fixed linters errors.
- [ ]  Deployed the project using netlify.
- [ ]  Updated the latest Microverse README.md file.

### CODE REVIEW

File: src/pages/Nav.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+          <NavLink className="item" to="/Missions">
+            Missions
+          </NavLink>
+        </li>
+        <li className="link-item">
+          <NavLink className="item" to="/Profile">
+            My Profile
+          </NavLink>
+        </li>
+      </ul>
```
Comment (tresorsawasawa) [inline src/pages/Nav.js:31]: - [ ] As required for the project and from your template design, kindly ensure that the **navigation option** of the **active page** is indicated with an **underline**. Right now, no interaction styling is added to all the navigation options.

File: src/pages/Profile.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+          {
+          rockets.map((rocket) => (
+            rocket.reserved ? (
+              <div key={rocket.rocket_id} className="reserved-item reserved-rocket-item">
+                {rocket.rocket_name}
+              </div>
+            ) : ''
+          ))
+          }
+
```
Comment (tresorsawasawa) [inline src/pages/Profile.js:56]: - [ **OPTIONAL** ] What do you think if the `My Profile` section can have a placeholder default message when the "My Missions" or "My Rockets" lists are empty (**no missions joined or no rockets reserved**) to make them look meaningful and better.

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Oklukeok and @amare1990 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow Gitflow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/Profile.js (hunk 2 above)
Comment (amare1990) [reply src/pages/Profile.js]: Thanks @tresorsawasawa for the clear code review. I have fixed the optional one too.

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi team 👋, 

You have done a great job implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**